fetch.bs

```diff
@@ -2213,6 +2281,90 @@ clearly stipulates that <a>connections</a> are keyed on
 <!-- See https://github.com/whatwg/fetch/issues/114#issuecomment-143500095 for when we make
      WebSocket saner -->
 
+<p>The requirements for <dfn export id="concept-record-connection-timing-info">recording connection
```

Comment:
This shouldn't be exported. Also doesn't need an ID (it'll get one assigned automatically).